exo-setup/src/docker-setup.ls

```diff
@@ -32,17 +32,19 @@ class DockerSetup
       links: @_get-docker-links!
       environment: @_get-docker-env-vars!
       depends_on: @_get-service-dependencies!
-    for dependency, dependency-config of @service-config.dependencies
-      docker-config[dependency + dependency-config.dev.version] = @_get-service-dependency-docker-config dependency, dependency-config.dev
+    if @service-config.dependencies
```

Comment:
I think without that if statement, it returns `Cannot read property 'length' of undefined`